Update advanced-config.md (#1087)
signalfx/splunk-otel-java: docs/advanced-config.md

| `splunk.metrics.endpoint`               | `SPLUNK_METRICS_ENDPOINT`               | `http://localhost:9943` | Experimental | The SignalFx metrics endpoint to connect to.  Setting this will override the `splunk.realm` property.                                                                                                                                                         |
| `splunk.metrics.export.interval`        | `SPLUNK_METRICS_EXPORT_INTERVAL`        | `30s`                   | Experimental | The interval between pushing metrics.  <details><summary>Format</summary>Durations can be of the form `{number}{unit}`, where unit is one of `ms`, `s`, `m`, `h`, `d`. If no unit is specified, milliseconds is the assumed duration unit.</details>          |
| `splunk.metrics.implementation`         | `SPLUNK_METRICS_IMPLEMENTATION`         | `micrometer`            | Experimental | The metrics implementation used by the agent. Valid values are `micrometer` and `opentelemetry`.                                                                                                                                                              |

> | `splunk.metrics.force_full_commandline` | `SPLUNK_METRICS_FORCE_FULL_COMMANDLINE` | `false`                 | Experimental | Adds the full command line as a resource attribute for all metrics. Maximum length is 255 characters. Commands longer than 255 characters are truncated.                                                                                                       |

Reviewer: That kinda sounds like the max length of the full command line is 255 characters. WDYT about `If false, commands longer than 255 characters are truncated`?

Author: Like it! Could you make the suggestion here? I'll merge it.

Reviewer: ```suggestion
| `splunk.metrics.force_full_commandline` | `SPLUNK_METRICS_FORCE_FULL_COMMANDLINE` | `false`                 | Experimental | Adds the full command line as a resource attribute for all metrics. If false, commands longer than 255 characters are truncated.                                                                                                       |
```
